### ISSUE TITLE
[IOS A11y] Scrollable with label is not focusable

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#86499]
b/192430197

## Steps to Reproduce

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      debugShowCheckedModeBanner: false,
      home: Scaffold(
        body: ListView(
          children: [
            Text('text'),
            MergeSemantics(
              child: Scrollable(
                viewportBuilder: (_, __) => Text('scrollable'),
              )
            ),
            MergeSemantics(
                child: Container(height: 100, child: ListView(
                  children: [ Text('scrollable')],
                ))
            ),
          ],
        ),
      ),
    );
  }
}
```

1. turn on voice over
2. the last scrollable is not focusable
